client/src/featureform/serving.py

```diff
@@ -609,6 +671,26 @@ def from_dataframe(dataframe, include_label_timestamp):
     def pandas(self):
         return self._dataframe
 
+    def wait(self, timeout=None):
```

Comment:
Will this error when you have:
`client.training_set(......).batch(....).wait()`

Would batch() get executed first?

Reply:
batch would, but batch just converts the stream into a batch stream. It still returns self so wait work work the same way